config/simplecontroller.json

```diff
@@ -0,0 +1,100 @@
+{
+    "Rate": 1,
+    "Actions": [
+      {
+        "ActionId": "simplecontroller.reload",
```

Comment:
I think we should not allow the user to choose the implementation details for an action.
It's the controller implementation that decides which action it supports by exporting their identifiers.

`SimpleController` should export a list of `ActionIds` that will map to functions according to its implementation.

This can become simply `"ActionId": "Reload"`.

Reply:
I see, my initial idea was to do this with `reflect` but at the end of the day it was going to bring unnecessary complexity. One way or another for every action you add to `SimpleController` you will need to update available functions. I will update this accordingly.

Reply:
Yeah that's totally fine.